src/android/BluetoothLePlugin.java

```diff
@@ -2905,6 +2905,9 @@ public void onScanResult(int callbackType, ScanResult result) {
           JSONObject returnObj = new JSONObject();
 
           addDevice(returnObj, result.getDevice());
+          if(result.getScanRecord().getDeviceName() != null){
+            addProperty(returnObj, keyName, result.getScanRecord().getDeviceName().replace("\0", ""));
```

Comment:
What's the replace needed for? Perhaps this should be opt in since it changes the existing behavior